### PR TITLE
Chapter 8: Basic login

### PR DESCRIPTION
**Pull request by hnamphamvo**
image

### CODE REVIEW

**Comment by hnamphamvo** [comment]
ready

**Comment by hnamphamvo** [comment]
ready

**Comment by hnamphamvo** [comment]
ready

File: app/assets/javascripts/sessions.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (phamvanbk11) [inline app/assets/javascripts/sessions.coffee]: Xóa file này đi

**Comment by phamvanbk11** [comment]
commented

**Comment by hnamphamvo** [comment]
ready

File: app/assets/javascripts/sessions.coffee (hunk 1 above)
Comment (hnamphamvo) [reply app/assets/javascripts/sessions.coffee]: Dạ đã xóa ạ.

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**